mb/mb/hashes.py

```diff
@@ -51,7 +51,9 @@ def __init__(self, basename, src_dir=None, dst_dir=None,
         self.dst_dir = dst_dir
 
         self.dst_basename = '%s.size_%s' % (self.basename, self.size)
-        self.dst = os.path.join(self.dst_dir, self.dst_basename)
+        self.dst = None
+        if dst_dir:
```

Comment:
shouldnt this be self.dst_dir ?

Reply:
they are equal (see line 51), I don't see difference